ng/src/web/components/form/formgroup.js

```diff
@@ -48,38 +90,33 @@ const FormGroup = ({
     return null;
   }
 
-  className = classes('form-group', className);
-
-  if (title) {
-    let title_css = classes('col-' + titleSize, 'control-label');
-    if (titleOffset) {
-      title_css = classes('offset-' + titleOffset, title_css);
-    }
-
-    if (!is_defined(size)) {
+  if (title && !is_defined(size)) {
       size = 12 - titleSize - titleOffset;
-    }
-    title = <label className={title_css}>{title}</label>;
   }
 
-  let css = is_defined(size) ? 'col-' + size : '';
-
-  if (offset) {
-    css = classes('offset-' + offset, css);
-  }
   return (
-    <Layout
+    <FormGroupLayout
       flex
       align={['start', 'center']}
-      className={className}>
-      {title}
-      <Layout
+      className={className}
```

Comment:
Is passing the className to FormGroup still necessary?

Reply:
It has been during some changes I made before, but in the end, it's just a leftover. I removed it now.